tests/add-user-ui.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { test, expect } from "@playwright/test";
+
+test("has title", async ({ page }) => {
+  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
+
+  // Expect a title "to contain" a substring.
+  await expect(page).toHaveTitle("TS Trainee course");
+});
+
+test("Create button UI", async ({ page }) => {
+  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
+  const el = page.getByRole("button", { name: "Create" });
+  await expect(el).toBeVisible;
+  await expect(el).toHaveClass("btn btn-primary");
+  await expect(el).toHaveCSS("background-color", "rgb(13, 110, 253)");
+  // checking btn reaction to hover
+  await el.hover();
+  await expect(el).toHaveCSS("background-color", "rgb(11, 94, 215)");
+});
+
+test("Cancel button UI", async ({ page }) => {
```

Comment:
s, take into account, cancel button can exist for 'Add User' form, for 'Add Address' form, etc. Need more context